README.md

```diff
@@ -22,7 +22,7 @@ and packaging of the application is [Maven](https://maven.apache.org/).
 ### Dependencies
 
 Available dependencies are fetched from Maven Central. Further dependencies (not available in Maven Central) are located in Kitodo/src/main/webapp/WEB-INF/lib.
-The only thing that is not shipped is a Tomcat specific servlet API. This should come along with the Tomcat distribution you are building against.
+Two things that are not shipped are a Tomcat specific servlet API and an ElasticSearch server. First thing should come along with the Tomcat distribution you are building against. Second, you need to download by yourself from [ElasticSearch Download Page](https://www.elastic.co/downloads/past-releases/elasticsearch-2-3-1) and start before Maven build. On the Windows system ElasticSearch starts after open elasticsearch.bat file from bin folder.
```

Comment:
If a running ElasticSearch 2.x instance is a minimum requirement then we should could two things: downgrade ElasticSearch inside TravisCI tests to minimum supported ElasticSearch version and update installation guide for 3.x even to ElasticSearch 2.x. Or we should raise ElasticSearch to a 5.1.1 version everywhere (README.md, installation instruction of 3.x, ...) .